packages/ramp-core/src/fixtures/appbar/nav-button.vue

```diff
@@ -43,10 +43,13 @@
 
 <script lang="ts">
 import { defineComponent } from 'vue';
-import { Vue } from 'vue-property-decorator';
+import DropdownMenuV from '@/components/controls/dropdown-menu.vue';
```

Comment:
is this import still required (`DropdownMenuV` should be registered to the app globally)?

Reply:
Import is not required. I removed it 👍